src/components/Match/Match.jsx

```diff
@@ -105,12 +106,12 @@ class RequestLayer extends React.Component {
       <div>
         <MatchHeader match={match} user={this.props.user} />
         <Tabs>
-          <Tab label="Overview">
+          <Tab label={strings.tab_overview}>
```

Comment:
we are likely to move to using the same TabBar component that we use for player pages, so this structure will likely change.  We'll probably define the pages in an array (like how we do player page list).